docs/Documentation/Configuration.md

```diff
@@ -0,0 +1,80 @@
+## Configuration
+
+Before you start with the configuration process, please ensure that the extension is properly installed on a working CiviCRM and that you have access to a functioning mailbox that you can and want to use for your MailboxMailing setup.
+
+### Compatability
+
+The MailboxMailing extension is compatible with every IMAP or POP3 mailserver. You need access to the mailbox via webmail or you can use basically every modern mailing program such as Gmail, Outlook, Apple icloud mail, yahoo and many others. You can also use Maildir oder Localdir to use the extension without having to connect through a mailserver like Outlook or Gmail.
+
+### Set up the Mailbox Connection
+
+To set up the Mailbox Connection go to *Administer* - *Administration Console* - *MailboxMailing settings.* Here you can *add Mailbox settings.* You can set a names for the groups of settings and create multiple groups for different mailboxes.
+
+In the process of setting up the Mailbox Connection you will be asked to enter the following information (only "Name" and "Protocol" are required):
+
+- **Name:** A name for this group of settings.
+    
+- **Server:** Name or IP address of the mail server machine you want to connect. Please note that the Server address is only required for IMAP and POP3 connections. For Maildir and Localdir, there is no server needed since these refer to local directories.
+    
+- **Username:** The username or email address used to access the mailbox.
+    
+- **Password:** The password used to access the mailbox
+    
+- **Protocol:** You can select the type of protocol to be used for polling (IMAP, Maildir, POP3, Localdir).
+    
+- **Source:** Folder or path to poll from. For IMAP/POP3, this is typically "INBOX." For Maildir or Localdir, this is the directory path.
+    
+- **Use SSL?:** Whether to use SSL (Secure Sockets Layer) for IMAP and POP3 or not. SSL ensures that the connection to the mail server is encrypted and secure. For Maildir and Localdir, there is no need for SSL settings, since they establish no network connection.
+    
+- **Sender Group:** The group of contacts allowed to send mailings using these MailSettings. Only emails from contacts in this group will be processed.
+    
+- **Recipient Group:** The group of contacts to receive mailings using these MailSettings. Only contacts in this group will be sent mailings.
+    
+
+The **Sender Group** and **Recipient Group** should consist of CiviCRM contact groups. This feature ensures that only authorized individuals can send emails and that specific groups receive the messages. This allows for effective management of incoming and outgoing emails.
```

Comment:
> should consist of CiviCRM contact groups

Actually, they *are* groups and need to be created before in order to be available in those settings fields.